flang/test/Lower/PowerPC/ppc-pwr10-vec-intrinsics.f90

```diff
@@ -28,9 +28,9 @@ end subroutine test_cvspbf16
 !CHECK-LABEL: @test_cvspbf16_
 !LLVMIR:  %1 = alloca <16 x i8>, i64 1, align 16
 !LLVMIR:  %2 = alloca <16 x i8>, i64 1, align 16
-!LLVMIR:  %3 = load <16 x i8>, ptr %2, align 16
+!LLVMIR:  %3 = load <16 x i8>, ptr %1, align 16
```

Comment:
Not you fault but these tests should not use hardcoded ssa value

Reply:
I agree, these tests need to be rewritten to not use hardcoded ssa values.